CHANGELOG.md

```diff
@@ -19,4 +19,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Now use the new `Base64Coder.encode` in order to encode to base64
 ### Removed
 - `cl.transbank.onepay.util.BASE64Encoder`
-- `cl.transbank.onepay.util.CharacterEncoder`
\ No newline at end of file
+- `cl.transbank.onepay.util.CharacterEncoder`
+
+## [1.2.0] - 2018-08-03
```

Comment:
If we haven't released the version 1.2.0 maybe we shouldn't put a date there

Reply:
And if we are making the release, then the date should be today, shouldn't it?

Reply:
Fixed